### ISSUE TITLE
[Bug] Emoji Skin Tone

### ISSUE THREAD

**Issue by sunkhaskasis** [lay295/TwitchDownloader#144]
When someone posts an emoji with a skin tone modifier, TD doesn't seem to recognize it. Here's a chat line rendered by TD:
![image](https://user-images.githubusercontent.com/19194315/107797464-07d18800-6d6c-11eb-84eb-6ad10cef89be.png)

And here's how it supposed to look like (screencapped from Twitch's VOD chat):
![image](https://user-images.githubusercontent.com/19194315/107797563-1f107580-6d6c-11eb-81f5-347530db8b3b.png)

Here's a json file containing this line. Look for "☝🏿 ZULUL POSTPONE ACTION":
https://i.nuuls.com/A1yks.json

**Comment by sunkhaskasis** [comment]
I just noticed that the file has since been deleted. Sorry, I'll provide a permanent mirror where the same issue is observed soon.

**Comment by sunkhaskasis** [comment]
A little late, but here's a different JSON file which contains an emoji with a skin color. Look for "👉🏿 🥱  👈🏿"
https://www.dropbox.com/s/6ox9ue23bcic243/%5B6-5-21%5D%20forsen%20-%20Games%20and%20shit%21.json?dl=1

**Comment by sunkhaskasis** [comment]
I fetched and compiled the latest version myself and can now confirm that the skin tone bug has been fixed as of the latest commit.
![image](https://user-images.githubusercontent.com/19194315/133928009-78e78649-f839-4a5a-a50f-935c7559c721.png)

However, it seems that the flag-related issue still persists:
![image](https://user-images.githubusercontent.com/19194315/133928041-8664cec8-365e-4109-8a57-9582f2568feb.png)

**Comment by lay295** [comment]
Yeah, I tried to fix it and then later realized I had a logic error that would prevent all the sequenced emojis from working. Emojis are hard :( damn zero width join modifiers.

But as my main goal was to fix the skin tones and the country flags so I just added it anyways
![image](https://user-images.githubusercontent.com/1060681/133939009-fb879247-5527-4663-b9c1-15e38602ce35.png)

The pirate flag is different than the country flags. But, I want to rewrite the rendering code one day since it really is a total hacked together mess. So maybe when I eventually do that I'll make sequenced emojis fully working.
